src/modules/local_position_estimator/BlockLocalPositionEstimator.cpp

```diff
@@ -426,29 +515,31 @@ void BlockLocalPositionEstimator::updateHome()
 	_altHome = alt;
 	_gpsAltHome += delta_alt;
 	_baroAltHome +=  delta_alt;
-	_lidarAltHome +=  delta_alt;
-	_sonarAltHome +=  delta_alt;
+	_visionHome(2) += delta_alt;
```

Comment:
@ecmnet 2-3cm difference probably wouldn't be that bad, but I'll go ahead and add the params